### PR TITLE
Fix core windows build that was failing when building coreclr

### PR DESCRIPTION
**Pull request by safern**
Coreclr build.cmd arguments order seems to matter in Windows. Since -PortableBuild=[true,false] is defined in config.json and passed all the way through run.exe the arguments after it (in Windows case -nopgooptimized) where passed into run.exe instead of being parsed by build.cmd. Causing this to failure saying -nopgooptimized is an invalid switch. 

Submitting the PR to see non-windows builds still work.

cc: @weshaggard

### CODE REVIEW

File: repos/coreclr.proj [hunk 1]
```diff
@@ -2,12 +2,12 @@
...
   <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.props))/dir.props" />
   <PropertyGroup>
-    <BuildArguments>$(Platform) $(Configuration) skiptests -PortableBuild=$(PortableBuild) </BuildArguments>
+    <BuildArguments>$(Platform) $(Configuration) skiptests</BuildArguments>
     <BuildArguments Condition="'$(SkipDisablePgo)' != 'true'">$(BuildArguments) -nopgooptimize</BuildArguments>
     <BuildArguments Condition="'$(OS)' != 'Windows_NT'">$(BuildArguments) msbuildonunsupportedplatform</BuildArguments>
     <BuildArguments Condition="'$(UseSystemLibraries)' == 'true'">$(BuildArguments) cmakeargs -DCLR_CMAKE_USE_SYSTEM_LIBUNWIND=TRUE</BuildArguments>
     <BuildArguments Condition="$(Platform.Contains('arm'))">$(BuildArguments) skipnuget cross -skiprestore cmakeargs -DFEATURE_GDBJIT=TRUE</BuildArguments>
-    <BuildCommand>$(ProjectDirectory)/build$(ShellExtension) $(BuildArguments) --</BuildCommand>
+    <BuildCommand>$(ProjectDirectory)/build$(ShellExtension) $(BuildArguments) -PortableBuild=$(PortableBuild) --</BuildCommand>
```
Comment (dagood) [inline repos/coreclr.proj]: Can you add this to `BuildArguments` instead, like `<BuildArguments>$(BuildArguments) -PortableBuild=$(PortableBuild)</BuildArguments>` on the line before this one? Just to keep things organized.

Comment (safern) [reply repos/coreclr.proj]: Sure. I can do that.

**Comment by dagood** [comment]
Seems like a safe change, but I'm not sure what's driving this PR. At the moment I think we'll want to wait until 2.1 ships to do any improvements to make source-build more buildable on Windows.

/cc @crummel @dseefeld @dleeapho

**Comment by weshaggard** [comment]
>Seems like a safe change, but I'm not sure what's driving this PR. At the moment I think we'll want to wait until 2.1 ships to do any improvements to make source-build more buildable on Windows.

@dleeapho  Can we merge the dev/release/2.1 branch into master and do work for my MQ source-build stuff there?

**Comment by safern** [comment]
Closing this PR since this work is going to be done in master.

**Pull request closed without merging**